app/scripts/modules/core/delivery/executionGroup/execution/execution.component.html

```diff
@@ -0,0 +1,115 @@
+        <span ng-if="::$ctrl.restartDetails">
+          Restarted by {{::$ctrl.restartDetails.restartedBy}} &mdash; {{::$ctrl.restartDetails.restartTime | timestamp}}
+        </span>
+        <span class="pull-right">Duration: {{$ctrl.execution.runningTimeInMs | duration}}</span>
+      </div>
+    </div>
+    <div class="execution-actions">
+      <a href
+         ng-if="::$ctrl.execution.isRunning"
+         analytics-on="click"
+         analytics-category="Pipeline"
+         analytics-event="Execution pause clicked"
+         ng-click="$ctrl.pauseExecution(); $event.stopPropagation();"
+         uib-tooltip="Pause execution">
+        <span class="glyphicon glyphicon-pause"></span>
+      </a>
+      <a href
+         ng-if="::$ctrl.execution.isPaused"
+         analytics-on="click"
+         analytics-category="Pipeline"
+         analytics-event="Execution resume clicked"
+         ng-click="$ctrl.resumeExecution(); $event.stopPropagation();"
+         uib-tooltip="Resume execution">
+        <span class="glyphicon glyphicon-play"></span>
+      </a>
+      <a href
+         ng-if="::!$ctrl.execution.isActive"
+         analytics-on="click"
+         analytics-category="Pipeline"
+         analytics-event="Execution delete clicked"
+         ng-click="$ctrl.deleteExecution(); $event.stopPropagation();"
+         uib-tooltip="Delete execution">
+        <span class="glyphicon glyphicon-trash"></span>
+      </a>
+      <a href
+         ng-if="::$ctrl.execution.isActive"
+         analytics-on="click"
+         analytics-category="Pipeline"
+         analytics-event="Execution cancel clicked"
+         ng-click="$ctrl.cancelExecution(); $event.stopPropagation();"
+         uib-tooltip="Cancel execution">
+        <span class="glyphicon glyphicon-remove-circle"></span>
+      </a>
+    </div>
+  </div>
+  <div ng-if="$ctrl.showingDetails" class="execution-graph">
+    <pipeline-graph ng-if="$ctrl.execution.parallel"
+                    execution="$ctrl.execution"
+                    on-node-click="$ctrl.toggleDetails(node)"
+                    view-state="$ctrl.viewState">
+
+    </pipeline-graph>
+  </div>
+
+  <div ng-if="$ctrl.showingDetails" class="execution-details-container">
+    <execution-details execution="::$ctrl.execution" application="::$ctrl.application" standalone="::$ctrl.standalone">
+    </execution-details>
+    <div class="permalinks">
+      <div class="permalinks-content">
+        <a target="_blank"
+           analytics-on="click"
+           analytics-category="Pipeline"
+           analytics-event="Execution source clicked"
+           ng-href="{{$ctrl.pipelinesUrl + $ctrl.execution.id}}">Source</a>
+        |
+        <a ng-href="{{::$ctrl.getUrl()}}"
+           analytics-on="click"
+           analytics-category="Pipeline"
+           analytics-event="Permalink clicked">Permalink</a>
+        <copy-to-clipboard text="{{::$ctrl.getUrl()}}" tool-tip="Copy permalink to clipboard"></copy-to-clipboard>
```

Comment:
We shouldn't one-time bind `$ctrl.getUrl()` (this is an existing bug, not something you introduced) - same as above on the `ng-href`